src/Visualize.ts

```diff
@@ -16,10 +16,7 @@ class Visualize implements Disposable, TextDocumentContentProvider {
   constructor(languageClient: LanguageClient, outputChannel: OutputChannel) {
     this.languageClient = languageClient;
     this.outputChannel = outputChannel;
-    this.disposables = [
-      commands.registerCommand("syntaxTree.visualize", this.visualize),
-      workspace.registerTextDocumentContentProvider("syntaxTree.visualize", this)
```

Comment:
Does it still work without registering the text document content provider?

Reply:
Oops! No, it did not. Restored missing line & tested to verify that visualize works.